Components/LandingPage/LandingContact.tsx

```diff
@@ -1,12 +1,58 @@
-import Image from "next/image";
+import React, { useState } from "react";
 
 const LandingContact = () => {
+  const [userData, setUserData] = useState({
+    name: "",
+    email: "",
+    message: "",
+  });
+
+  const postUserData = (event) => {
+    const { name, value } = event.target;
+    setUserData({ ...userData, [name]: value });
+  };
+
+  const submitData = async (event) => {
+    event.preventDefault();
+    const { name, email, message } = userData;
+
+    if (name &&   email && message) {
+      const res = await fetch(
+        // add your realtime database link here with userDataRecos.json extension
+        "https://gyan-934ec-default-rtdb.firebaseio.com/userDataRecords.json",
+        {
```

Comment:
Using database link is not a good practice. Please make a proper context in Context folder and then use firebase methods to send the contact form info to realtime database.